apis/swagger.yml

```diff
@@ -500,6 +500,29 @@ paths:
           required: true
       tags: ["Exec"]
 
+  /containers/{id}/resize:
+    post:
+      summary: "changes the size of the tty for a container"
+      operationId: "ContainerResize"
+      parameters:
+        - $ref: "#/parameters/id"
+        - name: "height"
+          in: "query"
+          description: "height of the tty"
+          type: "string"
+        - name: "width"
+          in: "query"
+          description: "width of the tty"
+          type: "string"
+      responses:
+        200:
+          description: "no error"
+        404:
```

Comment:
You returned 400(StatusBadRequest), and I am afraid you should add this in your swagger.yml to keep consistent.